test/morse_translator_test.rb

```diff
@@ -0,0 +1,26 @@
+require 'pry'
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/morse_translator.rb'
+
+class MorseTranslatorTest < Minitest::Test
+  def test_it_exists
+    morse_translator = MorseTranslator.new
+    assert_instance_of MorseTranslator, morse_translator
+  end
+
+  def test_translate
+    morse_translator = MorseTranslator.new
+    actual = morse_translator.translate("hello world")
+    expected = "......-...-..--- .-----.-..-..-.."
+    assert_equal expected, actual
+  end
+
+  def test_translate_capitals_and_numbers
```

Comment:
Would be good to see a test for capitals separately as well.

Reply:
Totally agree